lib/simulator-management.js

```diff
@@ -6,21 +6,30 @@ import _ from 'lodash';
 import log from './logger';
 import { tempDir, fs, mkdirp } from 'appium-support';
 import UUID from 'uuid-js';
+import { PLATFORM_NAME_IOS } from './desired-caps';
 
 
 const INSTALL_DAEMON_CACHE = 'com.apple.mobile.installd.staging';
 
+
+/**
+ * Capability set by a user
+ *
+ * @property {string} deviceName - A name for the device
+ * @property {string} platformVersion - The version of iOS to use
+ */
 /**
  * Create a new simulator with `appiumTest-` prefix and return the object.
  *
- * @param {object} caps - Capability set by a user. The options available are:
- *   - `deviceName` - a name for the device
- *   - `platformVersion` - the version of iOS to use
+ * @param {object} SimCreationCaps - Capability set by a user. The options available are:
+ * @property {string} platformName [iOS] - Platform name in order to specify runtime such as 'iOS', 'tvOS', 'watchOS'
  * @returns {object} Simulator object associated with the udid passed in.
  */
-async function createSim (caps) {
+async function createSim (caps, platformName = PLATFORM_NAME_IOS) {
```

Comment:
we could use `platform` for the parameter name, so then we can put the shortcut `{platform}`